### PR TITLE
fix(curriculum): specify array is returned from the function 

### LINKED ISSUE #43417: Sort an array alphabetically using the sort Method - need to specify the array should be returned from the function
## Affected page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method

## Your code

```
function alphabeticalOrder(arr) {
  // Only change code below this line
  arr.sort((a,b) => a === b ? 0 : a < b ? -1 : 1);

  // Only change code above this line
}
alphabeticalOrder(["a", "d", "c", "a", "z", "g"]);
```

## Expected behavior

The instructions on this page say, "Use the sort method in the alphabeticalOrder function to sort the elements of arr in alphabetical order."  Since sort happens in place, I didn't realize I need to return the value to pass the challenge, I kept thinking there was something wrong with my code.  Maybe just adding a line "return arr;" in the template code would make this more clear, or updating the instructions to specify the array needs to be returned.  Would be good to mention that sorting happens in place too, I only knew because I happened to look at the Mozilla documentation.

### PR DESCRIPTION
**Pull request by vamagithub**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #43417

### CODE REVIEW

**Review by Sembauke — DISMISSED** [review]

**Review by Pranav108 — DISMISSED** [review]

**Review by gikf — COMMENTED** [review]
I think return statement can be added as well (although it probably should be within the lines indicated for changing), but making in instructions explicit what function should return, would be more helpful for clarity.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ JavaScript's default sorting method is by string Unicode point value, which may
 
 # --instructions--
 
-Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order.
+Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order. The function will return a sorted array.
```
Comment (gikf) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method.md:41]:
```suggestion
Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order. The function should return a sorted array.
```

**Review by gikf — CHANGES_REQUESTED** [review]
I think "the sorted array" would be correct here, but as I'm not entirely sure I'll defer with it to others.

Below is small suggestion to make wording more consistent with curriculum.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ JavaScript's default sorting method is by string Unicode point value, which may
 
 # --instructions--
 
-Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order.
+Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order. The function should return a sorted array.
```
Comment (ieahleen) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method.md:41]:
```suggestion
Use the `sort` method in the `alphabeticalOrder` function to sort the elements of `arr` in alphabetical order. The function should return the sorted array.
```

a small nitpicking

Comment (vamagithub) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/sort-an-array-alphabetically-using-the-sort-method.md]: Done.

**Review by gikf — APPROVED** [review]
Looks good! :+1:

**Review by majestic-owl448 — APPROVED** [review]
LGTM :+1:

**Comment by gikf** [comment]
Hi @vamagithub. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**